test/integration/simple_example/controls/gcloud.rb

```diff
@@ -1,43 +1,34 @@
-# Copyright 2018 Google LLC
+# Copyright 2020 Google LLC
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#      http://www.apache.org/licenses/LICENSE-2.0
+#     https://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-project_id = attribute('project_id')
+require 'rest-client'
 
-control "gcloud" do
-  title "gcloud configuration"
-  describe command("gcloud --project=#{project_id} services list --available --format=json") do
-    its(:exit_status) { should eq 0 }
-    its(:stderr) { should eq '' }
-
-    let(:data) do
-      if subject.exit_status == 0
-        JSON.parse(subject.stdout)
-      else
-        {}
-      end
-    end
+gitlab_url = attribute('gitlab_url')
 
-    describe "enabled services" do
-      it "includes storage-api" do
-        expect(data).to include(
-          including(
-            "config" => including(
-              "name" => "storage-api.googleapis.com",
-            ),
-          ),
-        )
-      end
-    end
-  end
+control "gcloud" do
+	title "gitlab url"
+	describe "gitlab" do
+		it "is reachable" do
+			expect {
+				10.times do
+					unless host(gitlab_url.delete_prefix("https://"), port: 443, protocol: 'tcp').reachable?
+						puts "Gitlab is not reachable, retrying.."
+						sleep 10
+					end
+				end
+				RestClient::Request.execute(:url => gitlab_url, :method => :get, :verify_ssl => false)
```

Comment:
@morgante I seem to be hitting Lets Encrypt rate limits for`xip.io`, havent tried `nip.io` but I assume it will be the same, so I have turned this off for tests. lmk if you have any suggestions. 
```
"cert-manager/controller/orders "msg"="re-queuing item  due to error processing" "error"="error creating new order: acme: urn:ietf:params:acme:error:rateLimited: Error creating new order :: too many certificates already issued for: xip.io: see https://letsencrypt.org/docs/rate-limits/" "key"="default/gitlab-registry-tls-1738613401" 
```

Reply:
Could we use our own wildcard domain?

Fine to disable for now though.